app/javascript/vue/navigation/top_menu.vue

```diff
@@ -36,6 +36,7 @@
       <NotificationsFlyout
         v-if="notificationsOpened"
         :notificationsUrl="notificationsUrl"
+        :preferencesUrl="this.userMenu[1].url"
```

Comment:
This shouldn't be hardcoded, but found by the key, otherwise it will break if we change the order of items:
`this.userMenu.find((item) => item.name === I18n.t('users.settings.sidebar.account_nav.preferences')&.url`